arrows/mvg/applets/tests/CMakeLists.txt

```diff
@@ -4,3 +4,8 @@ add_test(NAME mvg:applets:track_features
   COMMAND kwiver track-features
             --config "${kwiver_test_data_directory}/track_features_testing.conf"
             "${kwiver_test_data_directory}/aphill_240_1fps_crf32.mp4")
+add_test(NAME mvg:applets:init_cameras_landmarks
+  COMMAND kwiver init-cameras-landmarks
+            --config "${kwiver_test_data_directory}/init_cameras_landmarks_testing.conf"
+            --video "${kwiver_test_data_directory}/aphill_240_1fps_crf32.mp4"
+            --tracks "${kwiver_test_data_directory}/tracks_light.txt")
```

Comment:
minor nitpick: we may want to add the missing newline at the end of this file, but let's wait for the CI to run first to see if there are any other issues.